apps/ledger-live-desktop/src/renderer/Default.tsx

```diff
@@ -133,21 +128,14 @@ const NightlyLayerR = () => {
 };
 const NightlyLayer = React.memo(NightlyLayerR);
 export default function Default() {
-  const location = useLocation();
-  const ref: React$ElementRef<any> = useRef();
   const history = useHistory();
   const hasCompletedOnboarding = useSelector(hasCompletedOnboardingSelector);
+
   useListenToHidDevices();
   useDeeplink();
   useUSBTroubleshooting();
   useProviders(); // prefetch data from swap providers here
 
-  // every time location changes, scroll back up
-  useEffect(() => {
-    if (ref && ref.current) {
-      ref.current.scrollTo(0, 0);
-    }
-  }, [location]);
```

Comment:
no longer needed?

Reply:
yeah the ref was never attached to an element